dataproc_jupyter_plugin/services/executor.py

```diff
@@ -88,12 +90,14 @@ async def get_bucket(self, runtime_env):
 
     async def check_file_exists(self, bucket, file_path):
         try:
-            cmd = f"gsutil ls gs://{bucket}/dataproc-notebooks/{file_path}"
-            await async_run_gsutil_subcommand(cmd)
-            return True
-        except subprocess.CalledProcessError as error:
-            self.log.exception(f"Error checking papermill file: {error.decode()}")
-            raise IOError(error.decode)
+            if not bucket:
+                raise ValueError("Bucket name cannot be empty")
+            bucket_name = storage.Client().bucket(bucket)
```

Comment:
The variable names here are backwards; the variable holding the name of the bucket is just called `bucket`, whereas the object representing the bucket is referenced by a variable named `bucket_name`.

Reply:
changed the variable names